architecture/additional_concepts/networking.adoc

```diff
@@ -22,6 +22,34 @@ recommend that you create a
 link:../core_objects/kubernetes_model.html#service[service], then interact with
 the service.
 
+== OpenShift DNS
+
+If you are running multiple services, such as frontend and backend services for use with multiple pods, in order for the frontend pods to communicate with the backend services, environment variables are created for such values as username, service-ip, etc. If the service is deleted and recreated, the IP address is assigned to a new environment variable, and requires the frontend pods to be recreated in order to pick up the updated values. Additionally, the backend service has to be created before any of the frontend pods to ensure that the service IP environment variable is generated properly and that the frontend pods can resolve to it.
```

Comment:
If the service is deleted and recreated, ~~the IP address is assigned to a new environment variable,~~ **a new IP address can be assigned to the service,** and requires the frontend pods to be recreated in order to pick up the updated values **for the service IP environment variable**. Additionally, the backend service has to be created before any of the frontend pods to ensure that the service IP ~~environment variable~~ is generated properly and that the **same can be provide to the frontend pods as an environment variable** ~~frontend pods can resolve to it~~ .